openscm/core/views.py

```diff
@@ -421,6 +423,48 @@ def values(self, v: np.ndarray) -> None:
             np.copyto(self._data, np.asarray(v))
         self._write()
 
+    @property
+    def time_points(self) -> np.ndarray:
+        """
+        Time points of this view
+        """
+        return self._time_points
+
+    @time_points.setter
+    def time_points(self, v: np.ndarray) -> None:
+        """
+        Time points of this view
+        """
+        self._time_points = v
+        self._version -= 1
+
+        def get_data_views_for_children_or_parameter(
```

Comment:
Why move it here?

Reply:
Because the child_views depend on the `time_points` and so have to all be updated whenever the `time_points` are updated.